git_machete/docs.py

```diff
@@ -314,6 +314,12 @@
           Any existing annotations are overwritten for the branches that have an opened PR; annotations for the other branches remain untouched.
           Equivalent to `git machete anno --sync-github-prs`.
 
+        <b>`checkout-pr` [PR_number]`:</b>
```

Comment:
```suggestion
        <b>`checkout-pr <PR-number>`:</b>
```

Reply:
Pls update the .rst docs under `docs/` as well (we'll need to keep them in sync until #211 is done :/)